src/pages/components/toggle.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Toggle'
+exampleKey: 'toggle'
+layout: '@layouts/ComponentLayout.astro'
+description: '設定をオンまたはオフに切り替えるためのコンポーネント'
+thumbnail: '/assets/images/components/thumbnail-toggle.svg.svg'
+---
+
+## Feature
+
+- 設定項目が2つの状態のみを持つ場合（例えば、設定の有効/無効）に適しています。
+- ユーザーが行った選択をすぐに反映させるシナリオに適しており、直感的に操作できるように設計されています。
+
+## Usage
+
+- `checked` と `onChange` を使用して値をコントロールして使用してください。
+  - `onChange` で通信を伴うなど待機時間が生まれる場合には、楽観的に状態を更新することが好まれます。
```

Comment:
These comments are by me, because I couldn't find the documentation by the designer on [figma](https://www.figma.com/file/ejIFAbw12HtoEZXcn01G5C/Component-Library?type=design&node-id=1612-7113&mode=design&t=jJGuL2UHEmokA5m0-0) (and related notion link).
Please update or give me advice to brush up, as I'm not yet fully understand Ubie's design principal. 🙏

Reply:
Thanks! I'll add an Issue to brush up on this content later.